cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/ConsoleSettingsHttpHandler.java

```diff
@@ -63,7 +61,7 @@ public ConsoleSettingsHttpHandler(ConsoleSettingsStore store) {
   @Path("/")
   @GET
   public void get(HttpRequest request, HttpResponder responder) throws Exception {
-    String userId = Objects.firstNonNull(SecurityRequestContext.getUserId(), "");
+    String userId = "";
```

Comment:
nit: Add a comment on the history and why it is not done in the right way (because customers are used to old behavior)